### PR TITLE
29787 text wrap

### PR DESCRIPTION
**Pull request by dletorey**
#### Summary

Updated the BCD for `text-wrap` to include separate values as different browsers are releasing different values at different times

#### Test results and supporting details

Tested in Firefox Nightly and Chromium browsers

#### Related issues

- [Implement text-wrap:balance](https://github.com/mdn/content/issues/29787)
- [Enable CSS text-wrap: balance for all channels](https://github.com/mdn/content/issues/30338)
- [Content PR](https://github.com/mdn/content/pull/30435)

### CODE REVIEW

File: css/properties/text-wrap.json [hunk 1]
```diff
@@ -64,6 +136,76 @@
...
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "stable": {
+          "__compat": {
+            "description": "<code>stable</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-wrap",
```
Comment (zfox23) [inline css/properties/text-wrap.json:149]:
```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-wrap#stable",
```

File: css/properties/text-wrap.json [hunk 2]
```diff
@@ -31,11 +33,81 @@
...
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "nowrap": {
+          "__compat": {
+            "description": "<code>nowrap</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-wrap",
```
Comment (zfox23) [inline css/properties/text-wrap.json:79]:
```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-wrap#nowrap",
```

File: css/properties/text-wrap.json [hunk 3]
```diff
@@ -31,11 +33,81 @@
...
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
         },
+        "balance": {
+          "__compat": {
+            "description": "<code>balance</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-wrap",
```
Comment (zfox23) [inline css/properties/text-wrap.json:44]:
```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-wrap#balance",
```

File: css/properties/text-wrap.json [hunk 4]
```diff
@@ -64,6 +136,76 @@
...
+              "experimental": true,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "wrap": {
+          "__compat": {
+            "description": "<code>wrap</code>",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-wrap",
```
Comment (zfox23) [inline css/properties/text-wrap.json:184]:
```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/text-wrap#wrap",
```

**Review by zfox23 — CHANGES_REQUESTED** [review]
suggestions to add anchors to BCD links

**Comment by dletorey** [comment]
> suggestions to add anchors to BCD links

Just added the link for the `pretty` value too

**Review by zfox23 — APPROVED** [review]
ty!

**Pull request merged**